Dockerfile

```diff
@@ -18,7 +18,8 @@ LABEL io.k8s.description="Base image for Red Hat Insights Compliance" \
 USER root
 RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
     yum install -y hostname shared-mime-info && \
-    yum clean all -y
+    yum clean all -y && \
+    gem update --system --install-dir=/usr/share/gems
```

Comment:
Similar comment from the backend. I think we'd be better of updating the base image than installing new rubygems here. These UBI images are intended to be used 'out of the box' and have their ruby environments optimized somehow. I'm not sure of the details, but I don't think this type of thing is recommended.